client/a8c-for-agencies/sections/marketplace/download-products-form/style.scss

```diff
@@ -0,0 +1,110 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "../mixins.scss";
+
+.download-products-form {
+	p {
+		font-size: 1rem;
+	}
+
+	.select-dropdown__container {
+		width: 100%;
+	}
+}
+
+.download-products-form__placeholder {
+	@include placeholder( --color-neutral-10 );
+
+	height: 43px;
+}
+
+.download-products-form__actions {
+	display: flex;
+	justify-content: flex-end;
+	margin: 42px -10px 0;
+
+	> * {
+		margin: 0 10px;
+	}
+}
+
+
+.download-products-form__pagination {
+	margin: 64px 0;
+}
+
+.download-products-form__action-items h4 {
+	font-size: 1.25rem;
+}
+
+.download-products-form__action-items h5 {
+	font-size: 1rem;
+}
+
+.download-products-form__action-items pre {
+	font-size: 0.875rem;
+	padding: 0;
+	margin-bottom: 0.6rem;
```

Comment:
This one can be improved using logical properties.

Reply:
Sorry for missing this.